INFO - 2025-05-17 16:03:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:03:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:03:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:03:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:06:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:06:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:06:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:06:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:11:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:36:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:37:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:44:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:44:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:44:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:45:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:45:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:45:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:46:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-17 16:46:40 --> ErrorException: Undefined variable $itemCount
in APPPATH\Views\dashboard\partials\common\layout.php on line 425.
 1 APPPATH\Views\dashboard\partials\common\layout.php(425): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $itemCount', 'C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\app\\Views\\dashboard\\partials\\common\\layout.php', 425)
 2 SYSTEMPATH\View\View.php(228): include('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\app\\Views\\dashboard\\partials\\common\\layout.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\View\View.php(244): CodeIgniter\View\View->render('dashboard/partials/common/layout', [], true)
 5 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('dashboard/newEnteryInWareHouse', [], true)
 6 APPPATH\Controllers\WhareHouseController.php(25): view('dashboard/newEnteryInWareHouse', [...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->add()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
INFO - 2025-05-17 16:47:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:47:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:54:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:54:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:55:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:56:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:58:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:59:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:59:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 16:59:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:03:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:27:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:27:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:27:20 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(95): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:27:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:27:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:27:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:27:50 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(95): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:27:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:29:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:29:32 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:29:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:29:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:29:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:29:56 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:29:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:38:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:38:15 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:38:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:38:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:38:33 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:38:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:45:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:45:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:46:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:46:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:46:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:46:30 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:46:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:47:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:47:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:47:32 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(92): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:47:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:48:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:48:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:48:16 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:48:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:49:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:49:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 17:49:46 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
INFO - 2025-05-17 17:49:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:56:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:56:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 17:58:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:00:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 18:00:50 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
CRITICAL - 2025-05-17 18:00:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:00:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'GS254434\', \'2025-05-17 18:00:50\', \'2025-05-17 18:00:50\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'GS254434\', \'2025-05-17 18:00:50\', \'2025-05-17 18:00:50\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:00:50 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'GS254434\', \'2025-05-17 18:00:50\', \'2025-05-17 18:00:50\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'GS254434\', \'2025-05-17 18:00:50\', \'2025-05-17 18:00:50\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'GS254434\', \'2025-05-17 18:00:50\', \'2025-05-17 18:00:50\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
INFO - 2025-05-17 18:01:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 18:01:41 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
CRITICAL - 2025-05-17 18:01:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:01:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'QI252669\', \'2025-05-17 18:01:41\', \'2025-05-17 18:01:41\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'QI252669\', \'2025-05-17 18:01:41\', \'2025-05-17 18:01:41\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:01:41 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'QI252669\', \'2025-05-17 18:01:41\', \'2025-05-17 18:01:41\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'QI252669\', \'2025-05-17 18:01:41\', \'2025-05-17 18:01:41\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'QI252669\', \'2025-05-17 18:01:41\', \'2025-05-17 18:01:41\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
INFO - 2025-05-17 18:02:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 18:02:45 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
CRITICAL - 2025-05-17 18:02:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:02:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'DL259951\', \'2025-05-17 18:02:45\', \'2025-05-17 18:02:45\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'DL259951\', \'2025-05-17 18:02:45\', \'2025-05-17 18:02:45\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:02:45 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'DL259951\', \'2025-05-17 18:02:45\', \'2025-05-17 18:02:45\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'DL259951\', \'2025-05-17 18:02:45\', \'2025-05-17 18:02:45\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'DL259951\', \'2025-05-17 18:02:45\', \'2025-05-17 18:02:45\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
INFO - 2025-05-17 18:05:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-17 18:05:54 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `it...', 0)
#1 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `it...')
#2 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `it...')
#3 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `it...', Array, false)
#4 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\app\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert(Array)
#8 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
#9 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
#10 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\Users\PYTHAGORAS\Desktop\codeigniter projects\kkk\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAG...')
#13 {main}
CRITICAL - 2025-05-17 18:05:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:05:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'FC258549\', \'2025-05-17 18:05:54\', \'2025-05-17 18:05:54\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'FC258549\', \'2025-05-17 18:05:54\', \'2025-05-17 18:05:54\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
CRITICAL - 2025-05-17 18:05:54 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'FC258549\', \'2025-05-17 18:05:54\', \'2025-05-17 18:05:54\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'FC258549\', \'2025-05-17 18:05:54\', \'2025-05-17 18:05:54\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (\'blouses\', \'Box\', \'1\', \'FC258549\', \'2025-05-17 18:05:54\', \'2025-05-17 18:05:54\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `items_table` (`item_name`, `item_package_type`, `item_quantity`, `item_custo_serial`, `date_logged`, `updated_at`) VALUES (:item_name:, :item_package_type:, :item_quantity:, :item_custo_serial:, :date_logged:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\WhareHouseController.php(87): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\WhareHouseController->save_entry()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\WhareHouseController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\PYTHAGORAS\\Desktop\\codeigniter projects\\kkk\\public\\index.php')
INFO - 2025-05-17 18:16:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:41:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:41:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:42:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:43:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:46:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:47:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-17 18:48:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
